aspirateur/tests/textes/test_get_dossiers_legislatifs.py

```diff
@@ -0,0 +1,96 @@
+import os
+from datetime import date
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+
+HERE = Path(os.path.dirname(__file__))
+
+
+@pytest.yield_fixture
+def sample_file():
+    path = HERE / ".." / "sample_data" / "Dossiers_Legislatifs_XV.json"
```

Comment:
`HERE.parent`?

Reply:
Fixed in f1cd03424abf0cd13f2f0434abdb09f38c24bb30